### ISSUE TITLE
Support Webpack 5

### ISSUE THREAD

**Issue by Nitraatss** [Epimodev/svg-sprite-html-webpack#16]
Hello. Are you plannig to add webpack 5 support?
Right now i am getting error:
WARNING : `compilation.hooks.htmlWebpackPluginBeforeHtmlProcessing` is undefined
SvgSpriteHtmlWebpackPlugin must be declare after HtmlWebpackPlugin to works
WARNING : `compilation.hooks.htmlWebpackPluginBeforeHtmlProcessing` is undefined
SvgSpriteHtmlWebpackPlugin must be declare after HtmlWebpackPlugin to works

"path": "0.12.7",
"glob": "7.1.6",
"webpack": "5.1.3",
"webpack-cli": "4.1.0",
"html-webpack-plugin": "4.5.0",
"svg-sprite-html-webpack": "2.3.0",

```
    ...glob.sync('./src/*.html').map((htmlFile) => new HtmlWebpackPlugin({
      filename: path.basename(htmlFile),
      template: htmlFile,
      inject: true,
      compress: true,
      minify: {
        removeComments: true,
      },
    })),

    new SvgSpriteHtmlWebpackPlugin({
      includeFiles: [paths.src + '/img/sprite/*.svg'],
    }),
```

**Comment by Epimodev** [comment]
Hello @Nitraatss 

Unfortunately I don't use webpack anymore so I didn't plan to add webpack v5 support.

If you know webpack and are able to do a pull request to add v5 support it will help me a lot.
If not I can do it but I don't know when I will have enough time to do it.

**Comment by Epimodev** [comment]
Hi @Nitraatss 

I don't know if you're still waiting for webpack 5 support but I just published the version 2.4.0 which support it.
Kudos to @christianopaets who did the pull request.
